web/src/features/Dashboard/api/getSelectedAppClusterDashboard.ts

```diff
@@ -47,8 +47,7 @@ export const useSelectedAppClusterDashboard = ({
 }: {
   refetchInterval: number | false;
 }) => {
-  const { data: isHelmManagedResponse } = useIsHelmManaged();
-  const { isHelmManaged } = isHelmManagedResponse || { isHelmManaged: false };
+  const { data: isHelmManaged = false } = useIsHelmManaged();
```

Comment:
one thing that kind of bugs me about this is we sometimes have `use-` hooks that you have to pull info off of `{ data: ... }` and some like `useSelectedApp` that just return the data and nothing about the async stuff. I think that inconsistency is bad but not sure how to handle it yet. Maybe we should just pass the `useQuery` interface into `useSelectedApp` but it's composing data from two endpoints.